### DISCUSSION TITLE
Manually installation of on-boot-script doesn't work because "udm-boot.boostchicken.dev" is offline

### DISCUSSION THREAD

**Discussion by m-lan-at** [unifi-utilities/unifi-common#441] [General]
Hello Community,

I love the UDM-Pro on-boot-script and installed it on many UDM-Pro machines. I always installed it manually because it only works when I install it manually over the unifi-os shell. Now my problem is that the server "https://udm-boot.boostchicken.dev" is offline and I can't install the on-boot-script. 

When I'm executing the command "curl -fsL "https://raw.githubusercontent.com/unifi-utilities/unifios-utilities/HEAD/on-boot-script/remote_install.sh" | /bin/sh" it installs but the containers like pi-hole don't start and enter a container state improper. 

I cannot see the container via "podman ps" and cannot stop or start them because they are running already. I have fixed this because I have manually installed the on-boot-script as described here "https://github.com/unifi-utilities/unifios-utilities/tree/main/on-boot-script". But the server is currently offline.

Do you have any ideas?

Thx a lot, Robin

**Comment by m-lan-at** [comment]
Sorry for the double post. Here is the link to the issue: https://github.com/unifi-utilities/unifios-utilities/issues/442

Edit: Can be marked as "closed". Thx.

**Comment by boostchicken** [comment]
fixed
